public/locales/en/translation.json

```
@@ -1,42 +1,41 @@
 {
   "pageTitle": {
     "ipfs": "IPFS Share",
-    "add": "Add Files",
+    "add": "Share Files",
     "download": "Download Files"
   },
   "header": "Share",
   "footer": "Licensed <1>CC-BY 3.0</1> except as <3>noted</3>",
   "info": {
     "add": {
-      "title": "Share files directly from your device",
+      "title": "Share files using IPFS",
+      "subtitle": "Share directly from your device — no cloud needed!",
       "labelAdd": "Add your files",
-      "copyAdd": "Anyone you give the link to will be able to get the files. Don't add sensitive or copyrighted files.",
-      "labelSend": "Send the share link",
-      "copySend": "IPFS gives the files a unique fingerprint called a CID that is used to find them again on the network.",
-      "labelKeepPage": "Keep this page open",
-      "labelKeepDaemon": "Keep your IPFS daemon running",
-      "copyKeep": "You are sharing the content from your device. Your friends can help you co-host the content by also keeping the share link open in their browser."
+      "copyAdd": "Add files to share them directly from your device over the IPFS network. Anyone with the share link can download your files, so don't add sensitive or copyrighted material.",
+      "labelKeepPage": "Keep this page open!",
+      "labelKeepDaemon": "Keep your IPFS node running!",
+      "copyKeepPage": "Because you're sharing directly from your device, it's important to keep this page open until the recipient downloads the files. If you're sharing with a group, others in the group can help you co-host the files by also keeping the share page open in their browsers.",
+      "copyKeepDaemon": "Because you're sharing directly from your device, it's important to keep your IPFS node running until the recipient downloads the files. If you're sharing with a group, others in the group can help you co-host the files by keeping the share page open in their browsers.",
```

Not feeling too strong about this, but perhaps casually mentioning "IPFS Desktop" is a thing here?
```suggestion
      "copyKeepDaemon": "Because you're sharing directly from your device, it's important to keep your IPFS node running until the recipient downloads the files. If you're sharing with a group, others in the group can help you co-host the files by keeping the share page open in their browsers or pinning the content identifier (CID) to IPFS Desktop node.",
```

---

Being that this message is only for folks who are somehow running an IPFS node already (daemon or Desktop), good call that we can be a little more detailed here. (Also planning on introducing a teaser to download Desktop in the modal text, once we have modal explainers.)

```suggestion
      "copyKeepDaemon": "Because you're sharing directly from your device, it's important to keep your IPFS node running until the recipient downloads the files. If you're sharing with a group, anyone in the group can help co-host the files by keeping the share page open in their browser, or if they're running an IPFS node, pinning the content identifier (CID) from your share link to their node.",
```